### PR TITLE
Flattie/next round button

### PR DESCRIPTION
**Pull request by sibela46**

### CODE REVIEW

File: src/components/PlayerInteractions/PlayerInteractions.jsx [hunk 1]
```diff
@@ -4,28 +4,35 @@ import Prompt from '../shared/Prompt/Prompt';
...
 
 const cx = classNames.bind(styles);
 
 export class PlayerInteractions extends React.Component {
+    constructor(props) {
+        super(props);
+        this.requestNextRound = this.requestNextRound.bind(this);
+    }
+
+    requestNextRound() {
```
Comment (Ro4052) [inline src/components/PlayerInteractions/PlayerInteractions.jsx]: Not sure there is any need for this method, you could just call `this.props.requestNextRound` directly.

File: src/components/PlayerInteractions/PlayerInteractions.jsx [hunk 2]
```diff
@@ -4,28 +4,35 @@ import Prompt from '../shared/Prompt/Prompt';
...
+        this.props.requestNextRound();
```
Comment (tillywoodfield) [inline src/components/PlayerInteractions/PlayerInteractions.jsx]: This doesn't need to be a whole function, and you can then remove the constructor right?

Comment (tillywoodfield) [reply src/components/PlayerInteractions/PlayerInteractions.jsx]: Oh I just realised Ronan already said this nevermind

**Comment by tillywoodfield** [comment]
Maybe I've missed something but it looks like you might run into problems with the timeout on the server? If you press next round and then the timeout also goes next round?

**Comment by sibela46** [comment]
> Maybe I've missed something but it looks like you might run into problems with the timeout on the server? If you press next round and then the timeout also goes next round?

The timeout is reset on every new round though, right?

File: src/components/Dashboard/Dashboard.jsx [hunk 3]
```diff
@@ -31,13 +31,16 @@ export class Dashboard extends React.Component {
                         <Chat />
                     </div>
                     <div className={styles.middle}>
-                        <div className={styles.interactions}>
+                        {<div className={styles.interactions}>
```
Comment (tillywoodfield) [inline src/components/Dashboard/Dashboard.jsx]: What's this curly brace for?

Comment (sibela46) [reply src/components/Dashboard/Dashboard.jsx]: Nothing anymore 🙈

**Review by tillywoodfield — APPROVED** [review]

**Pull request merged**